src/app/code/community/Omise/Gateway/Model/Payment/Offsitecitipoints.php

```diff
@@ -0,0 +1,143 @@
+<?php
```

Comment:
If this is basically the same as Alipay (and maybe others) - I think it would be better to make a parent class containing all the shared stuff, then have the similar offsite methods extend from there. This would be easier to maintain and make it much easier to add new similar methods

Reply:
@jonrandy I think this as this is the first PR of @mayurkathale , I would leave as it is in current architecture.
The aim of this PR is to add payment method, for @mayurkathale to see how things works.

Future refactoring should be done once @mayurkathale is more familiar with the plugin.

Reply:
Sorry - I totally disagree. Copy and pasting some way is no way to get familiar with the plugin. Doing some real coding however, is